InMobi/src/main/java/com/mopub/mobileads/InMobiInterstitialAd.kt

```diff
@@ -144,6 +144,7 @@ open class InMobiInterstitialAd : BaseAd() {
                     })
 
             mInMobiInterstitial?.run {
+                setExtras(Utils.extras)
```

Comment:
Why it is needed? I forgot the reasoning behind it.

Reply:
It is for adding tp-name and tp-ver

Reply:
But in load flow do we want it?